internal/overlord/servstate/manager_test.go

```diff
@@ -46,9 +46,9 @@ import (
 )
 
 const (
-	shortOkayWait = 50 * time.Millisecond
-	shortKillWait = 100 * time.Millisecond
-	shortFailWait = 200 * time.Millisecond
+	shortOkayDelay = 50 * time.Millisecond
+	shortKillDelay = 100 * time.Millisecond
+	shortFailDelay = 100 * time.Millisecond
```

Comment:
Why was this changed?

Reply:
It's because the original `shortFailWait` included both the time from first SIGTERM to SIGKILL and the second wait after that (100+100=200). Now the code is such that "fail delay" only includes that second part (which I think is a bit clearer in any case). So the total wait time here wasn't changed.

Reply:
Ack, thanks.